shared/stats/package.json

```
@@ -0,0 +1,15 @@
+{
+  "private": true,
+  "name": "govuk-frontend-stats",
+  "description": "GOV.UK Frontend shared stats",
+  "main": "index.mjs",
+  "engines": {
+    "node": "^18.12.0",
+    "npm": "^8.1.0 || ^9.1.0"
+  },
+  "license": "MIT",
+  "dependencies": {
```

Does it break anything if we set these to `optionalDependencies`?

I try to split things into:

1. `dependencies` → Published or deployed (Heroku) packages
2. `devDependencies` → Development or test packages only
3. `optionalDependencies` → Nice to haves like JSDoc types

We added a [little workaround for Heroku](https://github.com/alphagov/govuk-frontend/commit/9a08a406359e8c761e23fd449b9c96ab097bff95) but without it these two `dependencies` would be deployed

```suggestion
  "optionalDependencies": {
```